src/DistortableImageOverlay.js

```diff
@@ -245,6 +247,115 @@ L.DistortableImageOverlay = L.ImageOverlay.extend({
 		return this._dragStartPoints[0].subtract(this._dragPoints[0]);
 	},
 
+	_calcCenterTwoCornerPoints: function (topLeft, topRight) {
+			var toolPoint = { x: "", y: "" };
+
+      toolPoint.x = topRight.x + (topLeft.x - topRight.x) / 2;
+			toolPoint.y = topRight.y + (topLeft.y - topRight.y) / 2;
+			
+			return toolPoint;
+	},
+
+	_isInsideOut: function() {
+			var c0P = this._map.latLngToLayerPoint(this._corners[0]);
+      var c1P = this._map.latLngToLayerPoint(this._corners[1]);
+      var c2P = this._map.latLngToLayerPoint(this._corners[2]);
+      var c3P = this._map.latLngToLayerPoint(this._corners[3]);
+      var c0PY = L.Util.formatNum(c0P.y, 2);
+      var c0PX = L.Util.formatNum(c0P.x, 2);
+      var c1PY = L.Util.formatNum(c1P.y, 2);
+      var c1PX = L.Util.formatNum(c1P.x, 2);
+      // var c2PY = L.Util.formatNum(c2P.y, 2);
+      // var c3PY = L.Util.formatNum(c3P.y, 2);
+      var c3PX = L.Util.formatNum(c3P.x, 2);
+      var c2PX = L.Util.formatNum(c2P.x, 2);
+			return (
+				(c0PX > c1PX && this._isUpsideDown()) || 	
+				(c0PX < c1PX && this._isRightSideUp()) || 	
+				(this._isSidewaysLeft() && c0PY > c1PY) ||
+				((this._isSidewaysRight() && c0PY < c1PY) && (c1PX > c3PX && c0PX > c2PX))
```

Comment:
Amazing! 👍

Reply:
Thank you that one was a really proud moment for me  😹